### PR TITLE
[LA-350] Use upstream RPC-MaaS (Kilo)

### PR DESCRIPTION
**Pull request by cloudnull**
This change converts our deployment of RPC MaaS from the old role method
to the new playbook method. This change will provide us access to a
faster and more flexible monitoring solution and allow all customers to
benifit from Monitoring as a Service using a unified code base.

The maas group_vars have been removed and the values of the old file
have been placed into the `user_osa_variables_defaults.yml` file. This
has been done to allow the rpc-maas playbooks to inherit from the user
variables in cases where the group vars not not available.

LA-350

(cherry picked from commit b05681bec8cad4674af3d25ff7edc681d09706fe)
Signed-off-by: Kevin Carter <kevin.carter@rackspace.com>

### CODE REVIEW

**Comment by mancdaz** [comment]
```
+ chmod 0440 '/etc/openstack_deploy/user_*_defaults.yml'
chmod: cannot access ‘/etc/openstack_deploy/user_*_defaults.yml’: No such file or directory
script returned exit code 1
```

**Comment by odyssey4me** [comment]
Yeah, the user_*_defaults.yml file was introduced in mitaka, so it does not apply to kilo/liberty.

File: scripts/deploy.sh [hunk 1]
```diff
@@ -113,6 +113,16 @@ which openstack-ansible || ./scripts/bootstrap-ansible.sh
 cd ${RPCD_DIR}/playbooks
 openstack-ansible -i "localhost," patcher.yml
 
+# set permissions and lay down overrides files
+chmod 0440 /etc/openstack_deploy/user_*_defaults.yml
+if [[ ! -f "${RPCD_OVERRIDES}" ]]; then
+  cp "${RPCD_DIR}/etc/openstack_deploy/user_rpco_variables_overrides.yml" "${RPCD_OVERRIDES}"
+fi
```
Comment (mancdaz) [inline scripts/deploy.sh]: this will probably also fail in kilo

**Comment by cloudnull** [comment]
Updated to remove the extra includes from the master backport.

**Comment by evrardjp** [comment]
recheck_cit_trusty_leapfrogupgrade_swift

**Comment by hughsaunders** [comment]
Added nova_console_port to user_variables

**Comment by cloudnull** [comment]
recheck_cit_all

**Comment by cloudnull** [comment]
recheck_cit_all

**Comment by evrardjp** [comment]
recheck_cit_trusty_leapfrogupgrade_swift

**Comment by evrardjp** [comment]
recheck_cit_trusty_leapfrogupgrade_swift

**Comment by evrardjp** [comment]
I'm babysitting this one

**Comment by evrardjp** [comment]
Blocked on ansible_ssh_host undefined during leapfrog. Will submit a patch.

**Comment by evrardjp** [comment]
Depends on https://github.com/rcbops/rpc-openstack/pull/2385

**Comment by cloudnull** [comment]
recheck_cit_trusty_leapfrogupgrade_swift

**Comment by evrardjp** [comment]
recheck_cit_trusty_leapfrogupgrade_swift

**Pull request merged**